.github/workflows/testing-linux.yml

```diff
@@ -75,4 +76,34 @@ jobs:
         yml: .codecov.yml
         fail_ci_if_error: true
 
+  ##################################################################################
+  # Build on ROCm
+  ##################################################################################
+  build-on-rocm:
+    runs-on: ubuntu-latest
+    container:
+      image: shoshijak/build-env-rocm:1.0
```

Comment:
The Dockerfile I wrote should be pushed to dbcsr's dockerhu, and not stay on mine